### ISSUE TITLE
JSON Logging

### ISSUE THREAD

**Issue by felixguendling** [motis-project/motis#221]
Example:

```json
{
    "time": "2022-03-08T09:32:20.685+00:00",
    "severity": "WARN",
    "logger": "dispatcher.cc:121",
    "msg": "target not found: /metrics"
}
```

**Comment by derhuerst** [comment]
It would be great if the format would follow one of the (many) conventions out there, including the question how to deal with additional fields.

**Comment by felixguendling** [comment]
I can't find something like a real standard / RFC or JSON logging. Which format would you prefer?
There's a lot of content about JSON logging there but everybody seems to name fields in a different way, have different formats for timestamps, etc.

I think this looks good to me: https://nodejs.org/en/blog/module/service-logging-in-json-with-bunyan/

**Comment by derhuerst** [comment]
> I can't find something like a real standard / RFC on JSON logging.

Yeah, me too. Nothing fully established.

> Which format would you prefer?

I like to use [pino](https://getpino.io/) in the Node.js world, which logs in a format very similar to the well-known bunyan.

I also found the [Google Cloud schema](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry).

**Comment by felixguendling** [comment]
Makes sense. Being compatible to bunyan and/or pino would have the benefit of being able to use their pretty-print tools. Sounds great! :rocket:
